services-isochrone/src/test/java/com/mapbox/api/isochrone/MapboxIsochroneTest.java

```diff
@@ -0,0 +1,354 @@
+    MapboxIsochrone.builder()
+      .coordinates(testPoint)
+      .addContoursMinutes(5,25,40)
+      .profile(testProfile)
+      .baseUrl(mockUrl.toString())
+      .build();
+  }
+
+  @Test
+  public void build_invalidAccessTokenExceptionThrown() throws ServicesException {
+    thrown.expect(ServicesException.class);
+    thrown.expectMessage(StringStartsWith.startsWith("Using the Mapbox Isochrone API requires setting a valid access token."));
+    MapboxIsochrone.builder()
+      .accessToken("")
+      .addContoursMinutes(5,25,40)
+      .coordinates(testPoint)
+      .profile(testProfile)
+      .baseUrl(mockUrl.toString())
+      .build();
+  }
+
+  @Test
+  public void build_contoursAndColorsAmountMismatchExceptionThrown() throws ServicesException {
+    thrown.expect(ServicesException.class);
+    thrown.expectMessage(StringStartsWith.startsWith("Number of color elements must match number of minute elements provided."));
+    MapboxIsochrone.builder()
+      .accessToken(ACCESS_TOKEN)
+      .addContoursMinutes(5,25,40)
+      .addContoursColors("6706ce")
+      .coordinates(testPoint)
+      .profile(testProfile)
+      .baseUrl(mockUrl.toString())
+      .build();
+  }
+
+  @Test
+  public void build_noQueryExceptionThrown() throws Exception {
+    thrown.expect(IllegalStateException.class);
+    thrown.expectMessage("Missing required properties: coordinates");
+    MapboxIsochrone.builder()
+      .accessToken(ACCESS_TOKEN)
+      .profile(testProfile)
+      .addContoursMinutes(5,25,40)
+      .baseUrl(mockUrl.toString())
+      .build();
+  }
+
+  @Test
+  public void
+  build_invalidCoordinatesQueryExceptionThrown() throws ServicesException {
+    thrown.expect(JsonSyntaxException.class);
+    MapboxIsochrone.builder()
+      .accessToken(ACCESS_TOKEN)
+      .profile(testProfile)
+      .coordinates(Point.fromJson("4444,4949"))
+      .addContoursMinutes(5,25,40)
+      .baseUrl(mockUrl.toString())
+      .build();
+  }
+
+  @Test
+  public void build_noCountourMinutesExceptionThrown() throws Exception {
+    thrown.expect(IllegalStateException.class);
+    thrown.expectMessage("Missing required properties: contoursMinutes");
+    MapboxIsochrone.builder()
+      .accessToken(ACCESS_TOKEN)
+      .profile(testProfile)
+      .coordinates(testPoint)
+      .build();
+  }
+
+  @Test
+  public void build_invalidCountourMinutesExceptionThrown() throws Exception {
+    thrown.expect(ServicesException.class);
+    thrown.expectMessage("A query with at least one specified minute amount is required.");
+    MapboxIsochrone.builder()
+      .accessToken(ACCESS_TOKEN)
+      .profile(testProfile)
+      .coordinates(testPoint)
+      .addContoursMinutes()
+      .build();
+  }
+
+  @Test
+  public void build_optionalParameters() throws Exception {
+    MapboxIsochrone client = MapboxIsochrone.builder()
+      .accessToken(ACCESS_TOKEN)
+      .coordinates(testPoint)
+      .profile(testProfile)
+      .addContoursMinutes(5,25,40)
```

Comment:
Given that you are using the fixtures, Request should match the one that is in the fixture request 
contours_minutes=5,30,55&contours_colors=6706ce,04e813,4286f4

- This is just for the ease of debugging if problems occur in the future

Reply:
Ok, just pushed a change to fix this.

Reply:
and then rebased